src/sdk/__contracts/abi/CounterAbi.ts

```diff
@@ -0,0 +1,36 @@
+export const CounterAbi = [
```

Comment:
Do we need this in source? We already have CounterAbi in tests

Reply:
ok removing

Reply:
done